src/Avalonia.Dialogs/ManagedFileChooserFilterViewModel.cs

```diff
@@ -1,10 +1,11 @@
 using System.Linq;
 using System.Text.RegularExpressions;
+using Avalonia.Dialogs.Internal;
 using Avalonia.Platform.Storage;
 
 namespace Avalonia.Dialogs
```

Comment:
Inconsistency: `InternalViewModelBase` got moved to `Internal` namespace. I think the same should happen to those view models.

Reply:
Yes, I can do this.
My point was to make "InternalViewModelBase" less visible, as it should be used by anybody. While other dialog view models can be used in control restyling. But it also makes sense to move them to this namespace all anyway, as it's not common to change these styles.

Reply:
I think we need to make those `RaiseAndSetIfChanged<T>` methods  to be `protected internal` then

Reply:
And rename `InternalViewModelBase` to `AvaloniaDialogsInternalViewModelBase`

Reply:
Done